### PR TITLE
verify s390x builder instructions

### PR DESCRIPTION
**Pull request by LakshmiRavichandran1**

### CODE REVIEW

File: multi-arch-builders/README.md [hunk 1]
```diff
@@ -127,22 +127,6 @@ ibmcloud is instance-create $NAME $VPC $ZONE $PROFILE $SUBNET --output json --im
      --user-data @fcos-s390x-builder.ign > out.json
```
Comment (LakshmiRavichandran1) [inline multi-arch-builders/README.md:127]: The argument to ssh keys '--keys $SSHKEY' (can be obtained using 'ibmcloud is keys') is missing. is this expected?

Comment (dustymabe) [reply multi-arch-builders/README.md:127]: In this case we are passing SSH keys via Ignition, so no need to use the cloud keys. See 

https://github.com/coreos/fedora-coreos-pipeline/blob/f3f6c5ebf7688b1f91ff3e29e1d215bceea2d727/multi-arch-builders/fcos-s390x-builder.bu#L19-L26

We can update that list as needed.

File: multi-arch-builders/README.md [hunk 2]
```diff
@@ -127,22 +127,6 @@ ibmcloud is instance-create $NAME $VPC $ZONE $PROFILE $SUBNET --output json --im
...
-and then log in:
-
-```bash
-NIC=$(jq --raw-output .primary_network_interface.id out.json)
-ibmcloud is floating-ip-update fcos-s390x-builder-backup --nic $NIC
-
-INSTANCE=$(jq --raw-output .id out.json)
-IP=$(ibmcloud is instance $INSTANCE --output json \
-     | jq -r '.primary_network_interface.floating_ips[0].address')
-echo "You can now SSH to core@${IP}"
```
[the remark this answers is not in the document]
Comment (dustymabe) [reply multi-arch-builders/README.md:141]: In this case we use two different floating IPs. One is the backup IP used for us to just verify the new machine is up and functioning before we take down the old machine and move the main IP to it.

So we run floating IP commands twice.

[the remark this answers is not in the document]
Comment (LakshmiRavichandran1) [reply multi-arch-builders/README.md:141]: thank you for explaining this to me. I am able to get a picture of it now.

File: multi-arch-builders/README.md (hunk 1 above)
Comment (LakshmiRavichandran1) [reply multi-arch-builders/README.md:127]: thank you for pointing me to this.

**Comment by LakshmiRavichandran1** [comment]
Closing this PR as the changes are no more valid.

**Pull request closed without merging**